README.md

```diff
@@ -155,12 +164,16 @@ documentation.XX.po file).
 Before being able to compile the documents, you need to have a working
 compilation toolchain which you can get by running:
 
+```sh
     $ sh ci/install_po4a.sh # install a patched version of po4a
```

Comment:
Is this comment still acurate? I thought we switched to an unmodified po4a.

This should be updated. (unrelated to this change)

Reply:
It's unmodified but still not available at the right version on most distros

Reply:
Yeah, we should just update the comment to drop that inaccuracy.